packages/electron/package.json

```diff
@@ -39,5 +93,14 @@
     },
     "peerDependencies": {
         "electron": "12 - 28"
```

Comment:
side note outside this review:
should we bumb electron version to 32?

Reply:
sure